docs/components/plugins/api/APISectionEnums.tsx

```diff
@@ -10,18 +10,37 @@ export type APISectionEnumsProps = {
   data: EnumDefinitionData[];
 };
 
+const sortByValue = (a: EnumValueData, b: EnumValueData) => {
+  if (a.defaultValue && b.defaultValue) {
+    if (a.defaultValue.includes(`'`) && b.defaultValue.includes(`'`)) {
+      return a.defaultValue.localeCompare(b.defaultValue);
+    } else {
+      const vA = parseInt(a.defaultValue, 10);
+      const vB = parseInt(b.defaultValue, 10);
+      return vA > vB ? 1 : -1;
```

Comment:
I saw a lot that when you compare two numbers you simply subtract one from another obtaining as a result `negative number`, `0` or `positive number` 🧙 
```suggestion
      return vA - vB;
```

Reply:
Interesting, I will check this locally before the merge, thank you! :)